app/code/Magento/Catalog/Api/CategoryListInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
can we add required dependency via composer and do not copy them and for modules where we need only few of files - just copy-paster them to new namespace?